neuralmonkey/readers/plain_text_reader.py

```diff
@@ -52,20 +57,17 @@ def t2t_tokenized_text_reader(encoding: str = "utf-8") -> PlainTextFileReader:
     to preserve the whitespace around weird characters and whitespace on weird
     positions (beginning and end of the text).
     """
-    alphanumeric_charset = set(
-        chr(i) for i in range(sys.maxunicode)
-        if (unicodedata.category(chr(i)).startswith("L")
-            or unicodedata.category(chr(i)).startswith("N")))
-
     def reader(files: List[str]) -> Iterable[List[str]]:
         lines = string_reader(encoding)
         for line in lines(files):
             if not line:
                 yield []
-            line = line.rstrip("\n")
+            # github.com/tensorflow/tensor2tensor/blob/master/
```

Comment:
no need to cite this just because of the `strip` line. Plus, you are citing the code in the `master` branch, which to my surprise still points to a correct location (but soon won't).